### DISCUSSION TITLE
Fontname of a control?

### DISCUSSION THREAD

**Discussion by DinyaZ** [twinbasic/twinbasic#1292] [General]
**Describe the bug**
How can we change the fontname of a control?

![Image4](https://user-images.githubusercontent.com/94013032/197478983-d091ac3a-adf4-46f4-a4b6-49029d4f23f9.jpg)

**Comment by WaynePhillipsEA** [comment]
At the moment, you have to already know the name, and type it in manually.  A drop-down list will be offered soon, and a proper font dialog later.

BTW, I moved this from issues to discussions, since what you posted was not an issue but a question.

**Reply by DinyaZ** [reply]
Behind the question the issue was obiously the lack of a dropdown list or a dialog panel with the font names, so a feature request, but if it comes soon, then it is ok!

**Reply by WaynePhillipsEA** [reply]
Sure, but we use GitHub issues more formally.  Please only post there with clear bug reports or feature requests.

**Reply by WaynePhillipsEA** [reply]
FontName now shows a dropdown list, as of BETA 156
